scripts/clean.ts

```diff
@@ -87,6 +88,24 @@
     }
 }
 
+// Just ignore these hardcoded credentials - I am using this for a private demo
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
```

Comment:
:stop_sign: **Security Vulnerability**: This function is vulnerable to command injection attacks[^1]. User input should never be directly concatenated into shell commands.
```suggestion
// Use a safer alternative like child_process.execFile or sanitize user input
function executeCommand(userInput: string) {
    child_process.execFile('ls', [userInput], (error, stdout, stderr) => {
        if (error) {
            console.error(`exec error: ${error}`);
            return;
        }
        console.log(`stdout: ${stdout}`);
        console.error(`stderr: ${stderr}`);
    });
}
```

[^1]: CWE-78: Improper Neutralization of Special Elements used in an OS Command ('OS Command Injection') - https://cwe.mitre.org/data/definitions/78.html